### DISCUSSION TITLE
What to report in your paper if you used active learning software?

### DISCUSSION THREAD

**Discussion by Rensvandeschoot** [asreview/asreview#979] [Show and tell]
In this thread, we will share tips and tricks what to report in your paper if you used active learning software for systematic reviewing.

**Comment by boer0107-zz** [comment]
Probably the most important item to report when you use active learning is your stopping rule. Contrary to a traditional screening process you will stop screening before you have seen all titles and abstracts. The stopping rule describes the (predefined) point at which you have stopped screening. For instance after having screened a certain amount/share of your dataset or after having a certain number of irrelevant records in a row. Check our full discussion on stopping rules: #557

**Comment by Rensvandeschoot** [comment]
As with using any type of software, always report the version you used!!!! With [65 releases](https://github.com/asreview/asreview/releases) (and still counting) it matters which version you used.

**Comment by bmkramer** [comment]
The updated PRISMA guidelines now also include a mention on reporting on autho (

Checklist point 8: Methods > Selection process: 

_Specify the methods used to decide whether a study met the inclusion criteria of the review, including how many reviewers screened each record and each report retrieved, whether they worked independently, and **if applicable, details of automation tools used in the process**._

In the expanded checklist, this is elaborated further:

_Recommendations for reporting in systematic reviews using automation tools in the selection process:
• Report how automation tools were integrated within the overall study selection process.
• If an externally derived machine learning classifier was applied (e.g. Cochrane RCT Classifier), either to eliminate records or to replace a single screener, nclude a reference or URL to the version used. If the classifier was used to eliminate records before screening, report the number eliminated in the PRISMA flow diagram as ‘Records marked as ineligible by automation tools’.
• If an internally derived machine learning classifier was used to assist with the screening process, identify the software/classifier and version, describe how it was used (e.g. to remove records or replace a single screener) and trained (if relevant), and what internal or external validation was done to understand the risk of missed studies or incorrect classifications.
• If machine learning algorithms were used to prioritise screening (whereby unscreened records are continually re-ordered based on screening decisions), state the software used and provide details of any screening rules applied_

[http://prisma-statement.org/PRISMAStatement/Checklist](http://prisma-statement.org/PRISMAStatement/Checklist)

**Comment by boer0107-zz** [comment]
You can also check our [list of systematic reviews](https://www.zotero.org/groups/4597652/asreview_public/collections/IWU8ATMK/tags/systematic_review/collection) which made use of ASReview. What did these researchers report (not necessarily following our recommendations)?
